test/tests/RideRatings.cpp

```diff
@@ -0,0 +1,168 @@
+#include <string>
+#include <gtest/gtest.h>
+#include <openrct2/core/Path.hpp>
+#include <openrct2/core/String.hpp>
+#include <openrct2/OpenRCT2.h>
+#include "TestData.h"
+
+extern "C"
+{
+    #include <openrct2/platform/platform.h>
+    #include <openrct2/scenario/scenario.h>
+}
+
+class RideRatings : public testing::Test
+{
+protected:
+    static const char * ExpectedRideRatings[];
```

Comment:
I assume you are suggesting I make it utf8? There is nothing other than ascii in this file.